backend/core/src/migration/1626207809474-unit-status-enum.ts

```diff
@@ -0,0 +1,18 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class unitStatusEnum1626207809474 implements MigrationInterface {
+    name = 'unitStatusEnum1626207809474'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "units" DROP COLUMN "status"`);
```

Comment:
This migration will result in data loss in existing DBs, please first create a new type, then use alter table / alter column as e.g. [here](https://stackoverflow.com/a/15655926/2429333)

Reply:
Good catch, thanks! Is the most recent version what you're looking for?

Reply:
Yes, thank you.